File: doc/Examples_SPL/Expresiones/Expresiones/PrintInfix/integerInfix.cs
using System;
using expresiones;


namespace expresiones
{
    /**
     * Clase que define IntegerInfix
     **/
    public class IntegerInfix : BinaryOperatorInfix, Integer
    {
        int constante;
        /**
         * Constructor
         * */
        public IntegerInfix(int constante)
        {
            this.constante = constante;
            
        }
        /**
         * Método que imprime en formato infijo
         * */
        void ExpressionInfix.printInfix()
        {
            Console.Write(constante);
        }
    }
}

File: trunk/doc/Examples_SPL/expressionsCaesarJ/src/printInfix/IntegerInfix.java
cclass printInfix.PrintInfix;

public cclass IntegerInfix extends ExpressionInfix & Integer {
	protected int constante;
	public IntegerInfix(int constante){
		this.constante=constante;
	}
	public void print(){
		System.out.print(constante);
	}

}
